pymatgen/analysis/quasirrho.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+
+
+"""
+A module to calculate free energies using the Quasi-Rigid Rotor Harmonic
+ Oscillator approximation. Modified from a script by Steven Wheeler.
+See: Grimme, S. Chem. Eur. J. 2012, 18, 9955
+"""
+
+__author__ = "Alex Epstein"
+__copyright__ = "Copyright 2020, The Materials Project"
+__version__ = "0.1"
+__maintainer__ = "Alex Epstein"
+__email__ = "aepstein@lbl.gov"
+__date__ = "January 5, 2021"
+__credits__ = "Steven Wheeler, Trevor Seguin, Evan Spotte-Smith"
+
+from math import isclose
+from typing import Union
+
+import numpy as np
+import scipy.constants as const
+
+from pymatgen.core.units import amu_to_kg
+from pymatgen.core.units import kb as kb_ev
+from pymatgen.io.gaussian import GaussianOutput
+from pymatgen.io.qchem.outputs import QCOutput
+
+# Define useful constants
+kb = kb_ev * const.eV  # pymatgen kb from ev/K to J/K
+c = const.speed_of_light * 100  # cm/s
+h = const.h  # Planck's constant J.s
+R = const.R / const.calorie  # Ideal gas constant cal/mol
+
+# Define useful conversion factors
+kcal2hartree = 0.0015936  # kcal/mol to hartree/mol
+
```

Comment:
Is there a way to define this conversion factor using `scipy.constants`? If not, just make sure you carry enough decimal places to not lose precision due to the very different magnitudes of Ha and kcal.